content/self-host/rustdesk-server-pro/installscript/Docker/_index.en.md

```diff
@@ -7,12 +7,12 @@ weight: 6
 
 Install Docker with this [guide](https://docs.docker.com/engine/install) to ensure its the most up to date!
 
-Run the following commands:
+Run the following commands (be aware this might clear any files in the folder you run this in):
 
 ```sh
 sudo docker image pull rustdesk/rustdesk-server-pro
-sudo docker run --name hbbs -v `pwd`:/root -td --net=host --restart unless-stopped rustdesk/rustdesk-server-pro hbbs
-sudo docker run --name hbbr -v `pwd`:/root -td --net=host --restart unless-stopped rustdesk/rustdesk-server-pro hbbr
+sudo docker run --name hbbs -v `pwd`:/data -td --net=host --restart unless-stopped rustdesk/rustdesk-server-pro hbbs
+sudo docker run --name hbbr -v `pwd`:/data -td --net=host --restart unless-stopped rustdesk/rustdesk-server-pro hbbr
```

Comment:
If you do this, the database will not be generated on `pwd`, since hbbs/hbbr write all files under /root

Reply:
https://github.com/rustdesk/rustdesk-server/blob/272a094fde17a378276c039b4fd3caefadfb2569/docker-classic/Dockerfile#L4

https://github.com/rustdesk/rustdesk-server/blob/272a094fde17a378276c039b4fd3caefadfb2569/docker/Dockerfile#L22

Our stupid code

Reply:
I tested it and it is /root for OSS and /data for Pro

Reply:
I double checked, Pro follows OSS. `rustdesk-server-pro` image uses /root, `rustdesk-server-pro-s6` image uses /data. Which one did you test?

Reply:
Rustdesk-server-pro

Literally ran the commands exactly as above.

With /root no files appeared, with /data files appeared.

It used to be /root but it's changed

Reply:
![image](https://github.com/rustdesk/doc.rustdesk.com/assets/71636191/c8ebd7c4-4bd9-466e-95df-eb463d026146)

Reply:
ok tested again and can only get it with /root apart from s6, possibly I didnt delete it, Ill change the wording

Reply:
changed